### DISCUSSION TITLE
MocaccinoOS beta

### DISCUSSION THREAD

**Discussion by joostruis** [mocaccinoOS/mocaccino#58] [General]
Hello everybody!

After many months of hard work we can finally tag our first beta release!
MOS comes in several flavours, pick whatever suits you best.

New in this release:

- Split firmware packages _(We've split the linux-firmware package into smaller packages to free up additional space.)_
- Kernel is upgraded to 5.15.25 (LTS)
- Luet version 0.22.7
- Virt-manager is now available in the desktop repository.

Notes for testers:

- Additional software can be installed through flatpack, docker or the [community repository](https://github.com/mocaccinoOS/community-repository).
- Please give us feedback by reporting bugs or sharing your ideas with us.
- Check out our [documentation pages](https://www.mocaccino.org/docs/)

<hr /><em>This discussion was created from the release <a href='https://github.com/mocaccinoOS/mocaccino/releases/tag/v0.1-beta'>MocaccinoOS beta</a>.</em>

**Comment by VolvoBirkenstock** [comment]
w00t!!!  This is very excellent news, @joostruis.

**Comment by 01fiore** [comment]
Très bonne nouvelle, bravo.
